spec/factories/groups.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :group do
+    category "MyString"
+boolean ""
```

Comment:
Inconsistent indentation detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.